src/Controls/src/Core/Compatibility/Handlers/Windows/FrameRenderer.cs

```diff
@@ -94,35 +92,15 @@ void UpdatePadding()
 		{
 			// We need this so the `Border` control will arrange and have a size
 			Control?.Arrange(new WRect(0, 0, finalSize.Width, finalSize.Height));
-
-			if (Element is IContentView cv)
-			{
-				finalSize = cv.CrossPlatformArrange(new Rect(0, 0, finalSize.Width, finalSize.Height)).ToPlatform();
-			}
-
 			return new global::Windows.Foundation.Size(Math.Max(0, finalSize.Width), Math.Max(0, finalSize.Height));
 		}
 
 		protected override global::Windows.Foundation.Size MeasureOverride(global::Windows.Foundation.Size availableSize)
 		{
-			if (Element is IContentView cv)
-			{
-				// If there's a border specified, include the thickness in our measurements
-				// multiplied by 2 to account for both sides (left/right or top/bot)
-				var borderThickness = (Element.BorderColor.IsNotDefault() ? FrameBorderThickness : 0) * 2;
-
-				// Measure content but subtract border from available space
-				var measureContent = cv.CrossPlatformMeasure(
-					availableSize.Width - borderThickness,
-					availableSize.Height - borderThickness).ToPlatform();
+			Control?.Measure(availableSize);
```

Comment:
I think we need to remove the `UpdatePadding` code from the windows `FrameRenderer` now because that will be applied at the xplat level now vs on the control itself